pages/api/prospect/[id].tsx

```diff
@@ -0,0 +1,12 @@
+import prisma from '../../../lib/prisma'
```

Comment:
Let's deal only with the "Modal" component for this pull-request, and then we will deal with the API, ok?